wafer/talks/tests/test_views.py

```diff
@@ -160,6 +160,86 @@ def test_track_and_language(self):
         self.assertNotContains(response, '<td colspan="4">\nNo talks accepted yet.\n</td>', html=True)
 
 
+class TalksListSortingTests(TestCase):
+    """Test the table sorting options.
+
+       This monkey-patches TALKS.languages for the same reason as the layout tests.
+       """
+
+    def setUp(self):
+        """Create talks and tracks"""
+        self.client = Client()
+        Talk.LANGUAGES = (('en', 'English'), ('de', 'German'))
+
+        author = create_user('author_a')
+
+        # We choose the names so that they sort differently from creation order
+        track2 = Track.objects.create(name="Test Track 2", order=2)
+        track1 = Track.objects.create(name="Test Track 1", order=1)
+
+        talk_d = create_talk('Talk D', ACCEPTED, user=author, talk_track=track1,
+                                 language='en')
+        talk_c = create_talk('Talk C', ACCEPTED, user=author, talk_track=track2,
+                                 language='de')
+        talk_b = create_talk('Talk B', ACCEPTED, user=author, talk_track=track1,
+                                 language='de')
+        talk_a = create_talk('Talk A', ACCEPTED, user=author, talk_track=track2,
+                                 language='en')
+
+    def tearDown(self):
+        """Ensure Talk monkey-patching doesn't leak"""
+        Talk.LANGUAGES = ()
+
+    def test_no_sorting(self):
+        response = self.client.get('/talks/')
+        # We check via find to avoid hardcoding too much HTML here
+        pos_talk_a = response.content.find(b'Talk A')
+        pos_talk_b = response.content.find(b'Talk B')
+        pos_talk_c = response.content.find(b'Talk C')
+        pos_talk_d = response.content.find(b'Talk D')
+        # Order should be D, C, B, A (id order)
+        self.assertGreater(pos_talk_a, pos_talk_b)
+        self.assertGreater(pos_talk_b, pos_talk_c)
+        self.assertGreater(pos_talk_c, pos_talk_d)
```

Comment:
Is it possible to make this assert closer to `self.assertEqual(D < C < B < a)`?

Reply:
That construction loses information about which specific item was out of order if the test fails.